server/startup/prerender.js

```diff
@@ -0,0 +1,15 @@
+import prerender from "prerender-node";
+import { WebApp } from "meteor/webapp";
+import { Logger } from "/server/api";
+
+export default function () {
+  if (process.env.PRERENDER_TOKEN && process.env.PRERENDER_HOST) {
+    prerender.set("prerenderToken", process.env.PRERENDER_TOKEN);
+    prerender.set("host", process.env.PRERENDER_HOST);
+    prerender.set("protocol", "https");
+    WebApp.rawConnectHandlers.use(prerender);
+    Logger.info("Prerender Initialization finished.");
+  } else {
+    Logger.info("Prerender Not Configured. Skipping");
```

Comment:
Tiniest of items, but generally I am against `info` logging when we **don't** do things